sdk/go/pulumi-language-go/testdata/projects/l2-resource-simple-component/go.mod

```diff
@@ -0,0 +1,12 @@
+module l2-resource-simple-component
+
+go 1.20
+
+require (
+	github.com/pulumi/pulumi/sdk/v3 v3.30.0
```

Comment:
Unrelated to this PR really, but how come we're generating with such an old version?